autograder/handgrading/views/handgrading_result_views.py

```diff
@@ -332,6 +332,15 @@ class ListHandgradingResultsView(AGModelAPIView):
                                                 'type': 'number', 'format': 'double'
                                             },
                                         }
+                                    },
+                                    'has_autograded_submissions': {
```

Comment:
This is just a bikeshedding question, but perhaps we could call this something like "has_handgradeable_submission" instead?

Reply:
Fixed